list/index.php

```diff
@@ -48,29 +48,25 @@
 														plan_status 
 												FROM plan 
 												JOIN degree ON plan.degree_id = degree.degree_id
-												WHERE user_id = ?
+												WHERE user_id = ? AND deleted_ts IS NULL
 												ORDER BY modified_ts DESC", [$_SESSION["user_id"]]);
-							
-							foreach ($plans as $plan) {
-								echo "<tr>";
-								echo "<td>" . $plan["plan_title"] . planStatusToHTML($plan["plan_status"]) . "</td>";
-								echo "<td>" . $plan["major"]  . " " . $plan["year"] . "</td>";
-								echo "<td>" . date(DATE_FORMAT, strtotime($plan["created_ts"])) . "</td>";
-								echo "<td>" . date(DATE_FORMAT, strtotime($plan["modified_ts"])) . "</td>";
+							for ($i = 0; $i<count($plans); $i++) {
```

Comment:
Instead of converting the foreach loop to a for loop, you can do this in PHP to get the value and the index:
`foreach ($plans as $i => $plan)`

Although it looks like you didn't end up needing the index anyway.

Reply:
Ah. I did not know that. I was going to use the iterator to mark what plan's id was but I ended up not needing it since you showed me the relatedTarget function.